pkg/dm-crypt/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM linuxkit/alpine:3683c9a66cd4da40bd7d6c7da599b2dcd738b559 AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out  \
+    alpine-baselayout \
+    cryptsetup \
+    e2fsprogs
+
+# Remove apk residuals
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM scratch
+ENTRYPOINT []
+WORKDIR /
+COPY --from=mirror /out/ /
+
+COPY crypto.sh /usr/bin/crypto
```

Comment:
This still has the `crypto.sh` -> `crypto` rename which neither @deitch nor me liked. Could you fix this?

Reply:
Well, it was actually @deitch who suggested to do it like this, see https://github.com/linuxkit/linuxkit/pull/3249#discussion_r243295417

Reply:
Guilty as charged. I just wanted it to be `crypto` in the final package, not `crypto.sh`. @rn wanted it to be consistent from source to final package. I don't mind it being `crypto.sh` if it is hidden, but it is exposed when you call it, hence the desire for `crypto` (without the `.sh` extension). Maybe we should just keep it `crypto` I both src and pkg?